app/controllers/questions_controller.rb

```diff
@@ -87,7 +87,7 @@ def questions_property
     when 'not_solved'
       QuestionsProperty.new('未解決のQ&A', '未解決のQ&Aはありません。')
     else
-      QuestionsProperty.new('全ての質問', '質問はありません。')
+      QuestionsProperty.new('全てのQ&A', '質問はありません。')
```

Comment:
Issueには明示されていないものの、変更の意図を考慮すると`Question`が一件もない時の文言もあわせて変更する必要がありそうに思いましたが、いかがでしょうか 👀 
```suggestion
      QuestionsProperty.new('全てのQ&A', 'Q&Aはありません。')
```

Reply:
@yocajii 
レビューありがとうございます🙇‍♂️

> Descriptionの内容などプルリクの手順は全体的に問題なさそうで、さすがですね ✨

ありがとうございます😊 皆さんの日報やDocsのおかげです！

> Issueには明示されていないものの、変更の意図を考慮するとQuestionが一件もない時の文言もあわせて変更する必要がありそうに思いましたが、いかがでしょうか 👀

ご提案ありがとうございます🙏 未解決のQ&Aや解決済みのQ&Aもyocajiiさんが提案して頂いた形になっていたので、[#12762f2](https://github.com/fjordllc/bootcamp/pull/5256/commits/12762f2cc5d1ff8143c617d920e74607e696467b)で修正しました！